07.ls_object/ls.rb

```diff
@@ -56,27 +56,27 @@ def adjust_width(file_array, max_num)
 def print_files_details(files_in_directory)
   stat_max_length = calc_max_length_of_file_stat(files_in_directory)
   files_in_directory.each do |file|
-    print file_detail(file, stat_max_length)
+    print build_file_detail(file, stat_max_length)
     puts file.file_name
```

Comment:
```suggestion
    puts build_file_detail(file, stat_max_length)
```

file_nameだけ特別扱いする必要はないのでは？と思いました